### DISCUSSION TITLE
op node not synchronized

### DISCUSSION THREAD

**Discussion by lixiubao4967** [ethereum-optimism/developers#577] [Node Operators]
my config is following, why the node not synchronized?

okj-admin@okj-op-node-a01:/data/opnode/simple-optimism-node$ grep -vE '^\s*#|^\s*$' .env
NETWORK_NAME=op-mainnet
NODE_TYPE=full
BEDROCK_SOURCE=download
OP_NODE__RPC_ENDPOINT=http://35.78.2.181:7001
OP_NODE__L1_BEACON=http://35.78.2.181:3500
OP_NODE__RPC_TYPE=basic
HEALTHCHECK__REFERENCE_RPC_PROVIDER=https://mainnet.optimism.io
OP_GETH__HISTORICAL_RPC=https://mainnet.optimism.io
OP_GETH__SYNCMODE=snap

okj-admin@okj-op-node-a01:/data/opnode/simple-optimism-node$ sudo docker compose logs healthcheck -f --tail 10
WARN[0000] /data/opnode/simple-optimism-node/docker-compose.yml: `version` is obsolete
healthcheck-1  | {"level":30,"time":1719529915375,"targetHeight":0,"referenceHeight":121965567,"heightDifference":121965567,"minBlockNumber":0,"msg":"latest block heights"}
healthcheck-1  | {"level":30,"time":1719529915381,"log":"172.18.0.4 - GET /metrics HTTP/1.1 200 - - 3.308 ms\n","msg":"server log"}
healthcheck-1  | {"level":30,"time":1719529916464,"blockNumber":0,"msg":"blocks are matching"}
healthcheck-1  | {"level":30,"time":1719529922678,"targetHeight":0,"referenceHeight":121965571,"heightDifference":121965571,"minBlockNumber":0,"msg":"latest block heights"}
healthcheck-1  | {"level":30,"time":1719529923891,"blockNumber":0,"msg":"blocks are matching"}
healthcheck-1  | {"level":30,"time":1719529930408,"log":"172.18.0.4 - GET /metrics HTTP/1.1 200 - - 3.683 ms\n","msg":"server log"}
healthcheck-1  | {"level":30,"time":1719529930462,"targetHeight":0,"referenceHeight":121965575,"heightDifference":121965575,"minBlockNumber":0,"msg":"latest block heights"}
healthcheck-1  | {"level":30,"time":1719529931539,"blockNumber":0,"msg":"blocks are matching"}
healthcheck-1  | {"level":30,"time":1719529937681,"targetHeight":0,"referenceHeight":121965579,"heightDifference":121965579,"minBlockNumber":0,"msg":"latest block heights"}
healthcheck-1  | {"level":30,"time":1719529938923,"blockNumber":0,"msg":"blocks are matching"}
healthcheck-1  | {"level":30,"time":1719529945375,"targetHeight":0,"referenceHeight":121965583,"heightDifference":121965583,"minBlockNumber":0,"msg":"latest block heights"}
healthcheck-1  | {"level":30,"time":1719529945424,"log":"172.18.0.4 - GET /metrics HTTP/1.1 200 - - 7.869 ms\n","msg":"server log"}
healthcheck-1  | {"level":30,"time":1719529946006,"blockNumber":0,"msg":"blocks are matching"}

**Comment by imtipi** [comment]
please recreate a thread here https://github.com/ethereum-optimism/developers/discussions
